src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private int base;
+    private int side;
+
+    public Rectangle(int base, int side, Color color) {
+        super(color);
+        this.base = base;
+        this.side = side;
+    }
+
+    @Override
+    public double getArea() {
+        return base * side;
+    }
+
+    @Override
+    public String getFigureInfo() {
+        return "Figure: Rectangle, area: " + getArea() + " sq. units, base: " + base
+                + " side " + side + " units, color: " + getColor();

```

Comment:
There is a missing colon after 'side'. It should be 'side: ' to maintain consistency in the output format.